packages/std/src/addresses.rs

```diff
@@ -0,0 +1,88 @@
+use schemars::JsonSchema;
```

Comment:
Fair enough to separate this out.

I wonder a bit about the balance between shorter files and fewer files. But yeah, that is a discussion for another time

Reply:
For me the rule of thumb is when it becomes hard to switch between implementation and test section in the file.

In this particular case I think it helps to sort by domain and get rid of "types" at some point.